Guard polyfilled class declarations with class_exists() so they don't collide
with symfony/polyfill-php84's stubs on PHP < 8.4.

--- a/src/polyfill.php
+++ b/src/polyfill.php
@@ -5,21 +5,21 @@
 use Twint\Sdk\Certificate\PemCertificate;
 use Twint\Sdk\Certificate\Pkcs8Certificate;
 
-if (PHP_VERSION_ID < 80200) {
+if (PHP_VERSION_ID < 80200 && !class_exists('SensitiveParameter', false)) {
     #[Attribute(Attribute::TARGET_PARAMETER)]
     final class SensitiveParameter
     {
     }
 }
 
-if (PHP_VERSION_ID < 80300) {
+if (PHP_VERSION_ID < 80300 && !class_exists('Override', false)) {
     #[Attribute(Attribute::TARGET_METHOD)]
     final class Override
     {
     }
 }
 
-if (PHP_VERSION_ID < 80400) {
+if (PHP_VERSION_ID < 80400 && !class_exists('Deprecated', false)) {
     #[Attribute(Attribute::TARGET_METHOD | Attribute::TARGET_FUNCTION | Attribute::TARGET_CLASS_CONSTANT)]
     final class Deprecated
     {
